packages/data/tests/unit/adapters/facts/bard-test.ts

```diff
@@ -597,7 +709,37 @@ module('Unit | Adapter | facts/bard', function(hooks) {
   });
 
   test('_buildURLPath', function(assert) {
-    assert.expect(2);
+    assert.expect(4);
+
+    assert.equal(
+      Adapter._buildURLPath({ ...EmptyRequest, table: 'tableName' }),
+      'https://data.naviapp.io/v1/data/tableName/all/',
+      '_buildURLPath throws an error for missing dateTime column'
+    );
+
+    const twoDateTime: RequestV2 = {
+      ...EmptyRequest,
+      columns: [
+        {
+          field: '.dateTime',
+          type: 'timeDimension',
+          parameters: { grain: 'all' }
+        },
+        {
+          field: '.dateTime',
```

Comment:
Is the missing table name intentional?

Reply:
No but the `EmptyRequest` defaults everything so it has `table: ''` and I thought that was fine